index.html

```diff
@@ -0,0 +1,155 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div class="hamburger">
+                <span class="bars hamburger-bars"></span>
+                <span class="bars hamburger-bars"></span>
+                <span class="bars hamburger-bars"></span>
+            </div>
+            <ul class="menu">
+              <li><a href="#">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Speaker</a></li>
+              <li><a href="#">Sponsors</a></li>
+              <li><a href="#">News</a></li>
+              <li><a href="#">Register Now</a></li>
+           </ul>
+        </nav>
+    </header>
+    <section class="hero_section">
+
+        <div class="content1">
+         <h2>"Register Now"</h2>
+         <h1>UNLOCK THE </br> POWER OF DATA </br> SCIENCE WITH AI</h1>
+         <p>Welcome to 'Driven into Data Science</br> Using AI 2023',a premier event dedicated</br>
+            to exploring the intersection of data</br>science and artificial intelligence.Join us<br/>
+            for an immersive experience that will<br/>empower you with the knowledge and<br/>
+            skills needed to thrive in the data driven<br/> world. 
+         </p>
+
+        </div>
+         
+        <div class="venue">
+            <h2><strong>2023.10.15(THU) ~ 16(FRI)</strong></h2>
+            <h4>@ Field lane of Ghana Art Center Nabi and more</h4>
+        </div>
+       
+       
+    </section>
+
+    <section class="benefits">
+    <div class="content2">
+        <h3>Key Benefits<hr></h3>
+    </div>
+
+    <div class="gathering">
+        <div class="function">
+           <img src="/images/networking.jpg" alt="">
+           <h2>Networking</h2>
+           <p>Connect with industry experts,<br/> fellow data scientist and AI<br/> enthusiasts.</p>
+        </div>
+
+        <div class="function">
+            <img src="/images/insight.png" alt="">
+            <h2>Insights</h2>
+            <p>Gain valuable insights from<br/> keynote speakers and panel<br/> discussions</p>
+        </div>
+
+        <div class="function">
+            <img src="/images/workshop.jpg" alt="">
+            <h2>Workshop</h2>
+            <p>Participate in hands-on <br/> workshops to sharpen your data<br/> science and AI skills.</p>
+        </div>
+
+        <div class="function">
+            <img src="/images/workshops.jpg" alt="">
+            <h2>Workshops</h2>
+            <p>Try creating your own work using<br/>open source license rather than<br/> watching at it.</p>
+        </div>
+
+        <div class="function">
+            <img src="/images/network.jpg" alt="">
+            <h2>Network</h2>
+            <p>Get opportunities to network<br/> with students and developers<br/> around the world.</p>
+        </div>
+        <span><a>SEE THE WHOLE PROGRAM</a></span>
```

Comment:
Kindly increase the `font-size` for the `SEE THE WHOLE PROGRAM` , use the right `font-type` for the whole of your project, and align the `<span><a>SEE THE WHOLE PROGRAM</a></span>` to the center as it is in the design you were given👍kindly stick to the design
![DataLive - Google Chrome 02_01_2024 12_23_27](https://github.com/Mabella29/testn/assets/102887817/f7d73ce0-77c2-4d1e-9823-af286f5d89fb)